scripts/build/termux_step_get_dependencies.sh

```diff
@@ -107,11 +116,17 @@ termux_step_get_dependencies() {
 				read DEP_ARCH DEP_VERSION DEP_VERSION_PAC <<< $(termux_extract_dep_info $PKG "${PKG_DIR}")
 				termux_force_check_package_dependency && continue
 			else
-				[ ! "$TERMUX_QUIET_BUILD" = true ] && echo "Building dependency $PKG if necessary..."
+				[ ! "$TERMUX_QUIET_BUILD" = true ] && echo "Building dependency '$PKG' if necessary..."
 			fi
 			termux_run_build-package
 		fi
-	done<<<$(./scripts/buildorder.py $(test "${TERMUX_INSTALL_DEPS}" = "true" && echo "-i") "$TERMUX_PKG_BUILDER_DIR" $TERMUX_PACKAGES_DIRECTORIES || echo "ERROR")
+	done < <(./scripts/buildorder.py $(test "${TERMUX_INSTALL_DEPS}" = "true" && echo "-i") "$TERMUX_PKG_BUILDER_DIR" $TERMUX_PACKAGES_DIRECTORIES || echo -n "CMD_ERROR")
+	if [[ -n "${PKG:-}" ]] || [ $return_value -ne 1 ]; then
+		echo "Failed to get build order." 1>&2
+		[[ -n "${PKG:-}" ]] && echo "Error type: '$PKG'" 1>&2
```

Comment:
It will cause slight speed hit (because it is a hot codepath during dependencies obtaining step), probably will be better to 
```suggestion
	if [[ -n "${PKG}" || $return_value -ne 1 ]]; then
		echo "Failed to get build order." 1>&2
		[[ -n "${PKG}" ]] && echo "Error type: '$PKG'" 1>&2
```